### PR TITLE
openpilot v0.9.7 release

### PR DESCRIPTION
**Pull request by adeebshihadeh**
Version 0.9.7 (2024-06-13)
========================
* New driving model
  * Inputs the past curvature for smoother and more accurate lateral control
  * Simplified neural network architecture in the model's last layers
  * Minor fixes to desire augmentation and weight decay
* New driver monitoring model
  * Improved end-to-end bit for phone detection
* Adjust driving personality with the follow distance button
* Support for hybrid variants of supported Ford models
* Fingerprinting without the OBD-II port on all cars

### CODE REVIEW

File: RELEASES.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+Version 0.9.7 (2024-06-11)
+========================
+* New driving model
```
Comment (adeebshihadeh) [inline RELEASES.md:3]: @haraschax can you add some info about the new model?
